### ISSUE TITLE
DataTable warning after update to Backpack 3.3

### ISSUE THREAD

**Issue by remipou** [Laravel-Backpack/Settings#58]
# Bug report
Got an alert with DataTable warning: table_id=crudTable Ajax Error

### What I did:
updated to Backpack 3.3

### What I expected to happen:
Show my settings table

### What I've already tried to fix it:
Followed the link in the alert

### Backpack, Laravel, PHP, DB version:
Backpack 3.3
Laravel 5.5.21
Php 7.1

**Comment by rvpalivoda** [comment]
405 Method Not Allowed

**Comment by cerelli** [comment]
even with the demo comes the error:
`DataTables warning: table id=crudTable - Ajax error. For more information about this error, please see http://datatables.net/tn/7`

**Comment by allam2002** [comment]
I have the same issue and when i checked the Log i found that it is looking for the controller in another directory

**Class App\Http\Controllers\Admin\SettingCrudController does not exist**
i don't know how to solve that issue

**Comment by jobith93** [comment]
Please help with the issue, same issue +1

**Comment by dev** [comment]
Same issue here.

**Comment by tabacitu** [comment]
Hi guys,

The PR that fixes this issue has been merged on Nov 22 - just tested it again and it fixes the problem.

However, as of today Backpack\Demo was NOT using that new version of Backpack\Settings. If you were having this problem with the demo, that was the problem, I've now pushed an update to the demo for that.

In case you're having this problem in your own projects, a ```composer update``` will fix it for you guys.

Cheers!
